notebooks/2_PrincipalCovariatesRegression.ipynb

```diff
@@ -83,23 +83,32 @@
     "(1-\\alpha){\\left\\lVert \\mathbf{Y} - \\mathbf{X}\\mathbf{P}_{XT}\\mathbf{P}_{TY}\\right\\rVert^2}.\n",
     "\\end{equation}\n",
     "\n",
-    "It is easier to minimize this by looking for a projection $\\tilde{\\mathbf{T}}$ in a latent space for which we enforce orthonormality, $\\tilde{\\mathbf{T}}^T\\tilde{\\mathbf{T}} = \\mathbf{I}$\n",
+    "When $\\alpha = 1$, $\\ell = {\\left\\lVert \\mathbf{X} - \\mathbf{X}\\mathbf{P}_{XT}\\mathbf{P}_{TX}\\right\\rVert^2}$. It is equivalent to  principal component analysis (PCA). When $\\alpha = 0$, $\\ell = {\\left\\lVert \\mathbf{Y} \\mathbf{X}\\mathbf{P}_{XT}\\mathbf{P}_{TY}\\right\\rVert^2}$. It is equivalent to ridge regression.\n",
+    "\n",
+    "It is easier to minimize this by looking for a projection $\\tilde{\\mathbf{T}}$ in a latent space if we enforce the column vectors in $\\tilde{\\mathbf{T}}$ to be orthonormal, i.e., $\\tilde{\\mathbf{T}}^T\\tilde{\\mathbf{T}} = \\mathbf{I}$. Note that $\\tilde{\\mathbf{T}}\\tilde{\\mathbf{T}}^T \\neq \\mathbf{I}$ since the size of  $\\tilde{\\mathbf{T}}$ is $n_{samples}\\times n_{PCA} $and $n_{samples} > n_{PCA}$.\n",
     "\n",
     "\\begin{equation}\n",
     "\\ell =\n",
-    "\\alpha {\\left\\lVert \\mathbf{X} - \\mathbf{X}\\mathbf{P}_{XV}\\mathbf{P}_{VX}\\right\\rVert^2}\n",
+    "\\alpha {\\left\\lVert \\mathbf{X} - \\mathbf{X}\\mathbf{P}_{X\\tilde{T}}\\mathbf{P}_{\\tilde{T}X}\\right\\rVert^2}\n",
     " +\n",
-    "(1-\\alpha){\\left\\lVert \\mathbf{Y} - \\mathbf{X}\\mathbf{P}_{XV}\\mathbf{P}_{VY}\\right\\rVert^2}.\n",
+    "(1-\\alpha){\\left\\lVert \\mathbf{Y} - \\mathbf{X}\\mathbf{P}_{X\\tilde{T}}\\mathbf{P}_{\\tilde{T}Y}\\right\\rVert^2}.\n",
     "\\end{equation}\n",
     "\n",
-    "$\\tilde{\\mathbf{T}}$ is the [whitened](https://en.wikipedia.org/wiki/Whitening_transformation) version of our earlier projection $\\mathbf{T}$."
+    "$\\tilde{\\mathbf{T}}$ is the [whitened](https://en.wikipedia.org/wiki/Whitening_transformation) version of our earlier projection $\\mathbf{T}$.\n",
+    "Recall what we saw in [the previous notebook](1_LinearMethods.ipynb).\n",
+    "\n",
+    "\\begin{align}\n",
+    "& \\quad \\mathbf{K} = \\mathbf{X}\\mathbf{X}^T = \\mathbf{U}_\\mathbf{K} \\mathbf{\\Lambda}_\\mathbf{K} \\mathbf{U}_\\mathbf{K}^T \\\\\n",
+    "& \\quad\\mathbf{T} = \\hat{\\mathbf{U}}_\\mathbf{K} \\hat{\\mathbf{\\Lambda}}_\\mathbf{K}^{1/2} \\\\\n",
+    "& \\quad \\tilde{\\mathbf{T}} = \\hat{\\mathbf{U}}_\\mathbf{K}\n",
+    "\\end{align}\n"
```

Comment:
What is the reasoning behind this addition?

Reply:
For beginners like me, it is likely to be confused by the difference between \tilde {T} and T. These formulas will help.

Reply:
I understand that, although I don't think this is the best way to do this.

Reply:
Do you think it is better to delete it? Or is there another way?

Reply:
These relationships are mentioned in the part of MDS. I have modified it. If you insist, I will remove it.

Reply:
Given that both @bhelfrecht and I have said that this should be omitted, please omit this.

Reply:
I will remove it.